src/scenes/Partners/Detail/Tabs/Projects/PartnerDetailProjects.tsx

```diff
@@ -0,0 +1,33 @@
+import { Grid, Typography } from '@mui/material';
+import { PartnerDetailsFragment } from '../../PartnerDetail.graphql';
+import { PartnerDetailProjectsTable } from './PartnerDetailProjectsTable';
+
+interface Props {
+  partner: PartnerDetailsFragment | undefined;
+}
+
+export const PartnerDetailProjects = ({ partner }: Props) => {
+  const projects = partner?.projects;
+
+  return (
+    <Grid item xs={12}>
+      <Typography variant="h3" paragraph>
+        Projects
+      </Typography>
```

Comment:
Duplicated by tab label...
```suggestion
```

Reply:
<img width="655" alt="Screenshot 2023-10-18 at 8 57 13 PM" src="https://github.com/SeedCompany/cord-field/assets/138262921/697fab87-805d-4671-a620-202da8771742">

True, but it is in the design...